<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GestionUtlisateurTournois</class>
 <widget class="QDialog" name="GestionUtlisateurTournois">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>781</width>
     <height>491</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>170</y>
       <width>391</width>
       <height>201</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="Gestion_Utilisateur">
        <property name="styleSheet">
         <string notr="true"> #Gestion_Utilisateur{
 background-color: lightblue;
    border-style: outset;
    border-width: 2px;
    border-radius: 6px;
    border-color: beige;
    font: bold 14px;
    min-width: 4em;
    padding: 6px;
}</string>
        </property>
        <property name="text">
         <string>Gestion des utilisateurs</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Gestion_Tournois">
        <property name="styleSheet">
         <string notr="true"> #Gestion_Tournois
{
 background-color: lightblue;
    border-style: outset;
    border-width: 2px;
    border-radius: 6px;
    border-color: beige;
    font: bold 14px;
    min-width: 4em;
    padding: 6px;
}</string>
        </property>
        <property name="text">
         <string>Gestion des tournois</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="Retour_Menu">
     <property name="geometry">
      <rect>
       <x>643</x>
       <y>30</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#Retour_Menu
 {
 background-color: lightblue;
    border-style: outset;
    border-width: 2px;
    border-radius: 6px;
    border-color: beige;
    font: bold 14px;
    min-width: 4em;
    padding: 6px;
}</string>
     </property>
     <property name="text">
      <string>Retour</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>231</width>
       <height>361</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(173, 216, 230)</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">l#abel {

    font: bold 22px;
  ;
    padding: 6px;
}</string>
      </property>
      <property name="text">
       <string>Gestion des utlisateurs</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>210</width>
        <height>281</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>CIN</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Cin">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);
text-decoration:none;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Nom</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Nom">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);
text-decoration:none;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Mot de passe </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Motdepasse">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);
text-decoration:none;</string>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Adresse</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Adresse">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);
text-decoration:none;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Role</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox">
           <property name="styleSheet">
            <string notr="true">#comboBox{
 background-color: lightblue;
    border-style: outset;
    border-width: 2px;
    border-radius: 6px;
    border-color: beige;
    font: bold 12px;
    min-width: 4em;
    padding: 5px;
}</string>
           </property>
           <item>
            <property name="text">
             <string>Admin</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QTableView" name="TabUtilisatuer">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>100</y>
       <width>521</width>
       <height>331</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="Retour_Utilisateur">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>84</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#Retour_Utilisateur
 {
 background-color: lightblue;
    border-style: outset;
    border-width: 2px;
    border-radius: 6px;
    border-color: beige;
    font: bold 14px;
    min-width: 4em;
    padding: 6px;
}</string>
     </property>
     <property name="text">
      <string>Retour</string>
     </property>
    </widget>
    <widget class="QComboBox" name="Tri_utilisateur">
     <property name="geometry">
      <rect>
       <x>588</x>
       <y>61</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> #Tri_utilisateur{
 background-color: lightblue;
    border-style: outset;
    border-width: 2px;
    border-radius: 6px;
    border-color: beige;
    font: bold 14px;
    min-width: 4em;
    padding: 6px;
}</string>
     </property>
     <item>
      <property name="text">
       <string>Tri par nom (ASC)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Tri par role</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="Rechercher_utilisateur">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>70</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);
text-decoration:none;</string>
     </property>
     <property name="placeholderText">
      <string>Rechercher ...</string>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>440</y>
       <width>491</width>
       <height>35</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QPushButton" name="Ajouter_Utilisateur">
        <property name="styleSheet">
         <string notr="true">#Ajouter_Utilisateur {
 background-color: lightblue;
    border-style: outset;
    border-width: 2px;
    border-radius: 6px;
    border-color: beige;
    font: bold 14px;
    min-width: 4em;
    padding: 6px;
}</string>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Modifier_Utilisateur">
        <property name="styleSheet">
         <string notr="true">#Modifier_Utilisateur {
 background-color: lightblue;
    border-style: outset;
    border-width: 2px;
    border-radius: 6px;
    border-color: beige;
    font: bold 14px;
    min-width: 4em;
    padding: 6px;
}</string>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Supprimer_Utilisateur">
        <property name="styleSheet">
         <string notr="true">#Supprimer_Utilisateur {
 background-color: lightblue;
    border-style: outset;
    border-width: 2px;
    border-radius: 6px;
    border-color: beige;
    font: bold 14px;
    min-width: 4em;
    padding: 6px;
}</string>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Afficher_Utilisateur">
        <property name="styleSheet">
         <string notr="true">#Afficher_Utilisateur {
 background-color: lightblue;
    border-style: outset;
    border-width: 2px;
    border-radius: 6px;
    border-color: beige;
    font: bold 14px;
    min-width: 4em;
    padding: 6px;
}</string>
        </property>
        <property name="text">
         <string>Afficher</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>110</y>
       <width>241</width>
       <height>361</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(173, 216, 230)</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>14</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;
text-decoration: underline;</string>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string>Gestion des tournois</string>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>228</width>
        <height>261</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>ID</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ID">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);
text-decoration:none;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Date</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dateEdit">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);
text-decoration:none;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Nombre d'équipes</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Nombre_equipe">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);
text-decoration:none;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Adresse</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Adresse_Tournois">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);
text-decoration:none;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QPushButton" name="Retour_tournois">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>84</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#Retour_tournois {
 background-color: lightblue;
    border-style: outset;
    border-width: 2px;
    border-radius: 6px;
    border-color: beige;
    font: bold 14px;
    min-width: 4em;
    padding: 6px;
}</string>
     </property>
     <property name="text">
      <string>Retour</string>
     </property>
    </widget>
    <widget class="QTableView" name="TabTournois">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>111</y>
       <width>511</width>
       <height>321</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="Tri_Tournois">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>71</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#Tri_Tournois {
 background-color: lightblue;
    border-style: outset;
    border-width: 2px;
    border-radius: 6px;
    border-color: beige;
    font: bold 14px;
    min-width: 4em;
    padding: 6px;
}</string>
     </property>
     <item>
      <property name="text">
       <string>Trier par ID</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Trier par Date</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Trier par nombre  d'équipes</string>
      </property>
     </item>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>440</y>
       <width>461</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <widget class="QPushButton" name="Ajouter_Tournois">
        <property name="styleSheet">
         <string notr="true">#Ajouter_Tournois {
 background-color: lightblue;
    border-style: outset;
    border-width: 2px;
    border-radius: 6px;
    border-color: beige;
    font: bold 14px;
    min-width: 4em;
    padding: 6px;
}</string>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Modifier_Tournois">
        <property name="styleSheet">
         <string notr="true">#Modifier_Tournois {
 background-color: lightblue;
    border-style: outset;
    border-width: 2px;
    border-radius: 6px;
    border-color: beige;
    font: bold 14px;
    min-width: 4em;
    padding: 6px;
}</string>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Supprimer_Tournois">
        <property name="styleSheet">
         <string notr="true">#Supprimer_Tournois {
 background-color: lightblue;
    border-style: outset;
    border-width: 2px;
    border-radius: 6px;
    border-color: beige;
    font: bold 14px;
    min-width: 4em;
    padding: 6px;
}</string>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Afficher_Tounrois">
        <property name="styleSheet">
         <string notr="true">#Afficher_Tounrois {
 background-color: lightblue;
    border-style: outset;
    border-width: 2px;
    border-radius: 6px;
    border-color: beige;
    font: bold 14px;
    min-width: 4em;
    padding: 6px;
}</string>
        </property>
        <property name="text">
         <string>Afficher</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
